Makefile

```diff
@@ -262,8 +262,18 @@ cluster-sync:
 cluster-up:
 	rm -rf local-dev-cluster
 	git clone https://github.com/sustainable-computing-io/local-dev-cluster.git --depth=1
-	cd local-dev-cluster && ./main.sh
+	cd local-dev-cluster && ./main.sh up
```

Comment:
So far we are going to use v0.0.0 version of local-dev-cluster instead of latest.
ok... this is not only a rfc pr.
if so, I suppose the teardown support will be supported from v0.0.1 of local-dev-cluster.
Hence the suggestion here is that we have a variable for local-dev-cluster version usage.